Testing/SystemTests/tests/analysis/complexShapeAbsorptionsTest.py

```diff
@@ -0,0 +1,44 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+from __future__ import (absolute_import, division, print_function)
+import systemtesting
+import mantid.simpleapi as mantid
+from mantid import config
+import os.path
+
+
+DIRS = config['datasearch.directories'].split(';')
```

Comment:
Do we need to do this? Will the algorithms not do the file searching for us?

Reply:
huh, apparently we don't need this, Im just used to doing that because its how all the powder systemtests are written